### ISSUE TITLE
Shall I add documentation for modifiers?

### ISSUE THREAD

**Issue by umurgdk** [iron/iron#550]
I was struggling to understand how to use that modifiers, and what are the possibilities. Finally got the point, and would like to add documentation for each modifier and also how to add your own modifiers. But currently rustdoc doesn't support to show documentation for trait implementations --where they defined-- (in modifiers.rs) So it seems like the only options is adding those documentation to module level docs. 

Before i go and write documentation into module level docs, I wanted to ask if this is ok or not. Or maybe do you think ironframework.io is better place for these kind of documentation?

**Comment by robinst** [comment]
Yes please, it's one of the most confusing parts of Iron IMO. (Disclaimer: I'm not a maintainer.)
